Gruntfile.js

```diff
@@ -443,7 +424,7 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test', [
     'clean:beforeBuild',
-    'jsbeautifier:html',
+    'htmllint',
     'lesslint',
```

Comment:
The `test` task is mainly to test js and are run with the `watch:js` task so maybe here we need another task for linting html and css files.